server/db.js

```diff
@@ -1,12 +1,13 @@
-import { Pool } from "pg";
-
+import pg from "pg";
```

Comment:
Are the changes in this file necessary? Seems like nothing really changed

Reply:
I have reverted the change